modules/cco-ccoctl-creating-individually.adoc

```diff
@@ -0,0 +1,124 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc
+
+[id="cco-ccoctl-creating-individually_{context}"]
+= Creating AWS resources individually
+
+If you need to review the JSON files that the `ccoctl` creates before modifying AWS resources, or if the process the `ccoctl` uses to create AWS resources automatically does not meet the requirements of your organization, you can create the AWS resources individually. For example, this option might be useful for an organization that shares the responsibility for creating these resources among different users or departments.
+
+Otherwise, you can use the `ccoctl aws create-all` command to create the AWS resources automatically.
+
+//to-do if possible: xref to modules/cco-ccoctl-creating-at-once.adoc for `create the AWS resources automatically`
+
+[NOTE]
+====
+By default, `ccoctl` creates objects in the directory in which the commands are run. To specify a directory, use the `--output-dir` flag. This procedure uses `_<path_to_ccoctl_output_dir>_` to refer to this location.
+
+Some `ccoctl` commands make AWS API calls to create or modify AWS resources. To place JSON files on the local file system instead, use the `--dry-run` flag. These JSON files can be reviewed or modified and then applied with the AWS CLI tool using the `--cli-input-json` parameters.
+====
+
+.Prerequisites
+
+* Extract and prepare the `ccoctl` binary.
+
+.Procedure
+
+. Generate the public and private RSA key files are used to set up the OpenID Connect provider for the cluster:
++
+[source,terminal]
+----
+$ ccoctl create key-pair
+----
++
+.Example output:
++
+[source,terminal,subs="+quotes"]
+----
+2021/04/13 11:01:02 Generating RSA keypair
+2021/04/13 11:01:03 Writing private key to /__<path_to_ccoctl_output_dir>__/serviceaccount-signer.private
+2021/04/13 11:01:03 Writing public key to /__<path_to_ccoctl_output_dir>__/serviceaccount-signer.public
+2021/04/13 11:01:03 Copying signing key for use by installer
+----
++
+Where `serviceaccount-signer.private` and `serviceaccount-signer.public` are the generated key files.
++
+This command also creates a private key that the cluster requires during installation in `/_<path_to_ccoctl_output_dir>_/tls/bound-service-account-signing-key.key`.
+
+. Create an OpenID Connect identity provider and S3 bucket on AWS:
++
+[source,terminal,subs="+quotes"]
+----
+$ ccoctl create identity-provider --name=__<name>__ --region=__<aws_region>__ --public-key=__<path_to_ccoctl_output_dir>__/serviceaccount-signer.public
```

Comment:
This command should be `$ ccoctl aws create-identity-provider --name=__<name>__ --region=__<aws_region>__ --public-key-file=__<path_to_ccoctl_output_dir>__/serviceaccount-signer.public`